File: sdkwork-backend-sdk-java/src/main/java/com/sdkwork/backend/model/PasswordResetResultVO.java
package com.sdkwork.backend.model;

public class PasswordResetResultVO {
    private String createdAt;
    private String updatedAt;
    private Boolean success;
    private String message;
    private Integer expiresIn;
    private String token;

    public String getCreatedAt() {
        return this.createdAt;
    }
    
    public void setCreatedAt(String createdAt) {
        this.createdAt = createdAt;
    }

    public String getUpdatedAt() {
        return this.updatedAt;
    }
    
    public void setUpdatedAt(String updatedAt) {
        this.updatedAt = updatedAt;
    }

    public Boolean getSuccess() {
        return this.success;
    }
    
    public void setSuccess(Boolean success) {
        this.success = success;
    }

    public String getMessage() {
        return this.message;
    }
    
    public void setMessage(String message) {
        this.message = message;
    }

    public Integer getExpiresIn() {
        return this.expiresIn;
    }
    
    public void setExpiresIn(Integer expiresIn) {
        this.expiresIn = expiresIn;
    }

    public String getToken() {
        return this.token;
    }
    
    public void setToken(String token) {
        this.token = token;
    }
}

File: sdkwork-backend-sdk-typescript/src/types/password-reset-result-vo.ts
/** å¯ç éç½®ååºè§å¾å¯¹è±¡ */
export interface PasswordResetResultVO {
  /** åå»ºæ¶é´ */
  createdAt?: string;
  /** æ´æ°æ¶é´ */
  updatedAt?: string;
  /** æ¯å¦æå */
  success?: boolean;
  /** æ¶æ¯ */
  message?: string;
  /** è¿ææ¶é´ï¼ç§ï¼ */
  expiresIn?: number;
  /** éç½®ä»¤ç */
  token?: string;
}
